packages/composables/src/useCartErrorParamsResolver/useCartErrorParamsResolver.ts

```diff
@@ -2,7 +2,7 @@ import { useCart } from "#imports";
 
 export type UseCartErrorParamsResolver = {
   resolveCartError: (errorObject: ErrorObjectParam) => {
-    params: { [key: string]: string | number | null } | null;
+    params: { [key: string]: string | number | null | undefined } | null;
```

Comment:
why not `?` then?
```suggestion
    params: { [key: string]?: string | number | null } | null;
```

Reply:
Are you sure that this is a valid type? `?` - throws type syntax error

Reply:
yeah, you might be right in that case, so let's leave it like this :)